src/Summoner/CLI.hs

```diff
@@ -41,6 +47,35 @@ summon = execParser prsr >>= runCommand
 runCommand :: Command -> IO ()
 runCommand = \case
     New opts -> runNew opts
+    ShowInfo opts -> runShow opts
+
+runShow :: ShowOpts -> IO ()
+runShow opts =
+    case opts of
+        -- show list of all available GHC versions
+        ShowGhc          -> putText $ unlines $ map showGhcVer universe
```

Comment:
I guess we can improve the outlook a bit. As it's a list we can add the arrow symbol (the one we used in git revision).
And also it would be nice to shoe it in reversed order (from latest to oldest).